docs/content/en/docs/references/api/grpc.md

```diff
@@ -942,6 +1016,18 @@ Enum for Suggestion codes
 | OPEN_ISSUE | 900 | Open an issue so this situation can be diagnosed |
 
 
+
+<a name="proto.TesterType"></a>
+
+### TesterType
+Enum indicating test tools used
+
+| Name | Number | Description |
+| ---- |:------:| ----------- |
+| UNKNOWN_TEST_TYPE | 0 | Could not determine Test Type |
+| UNIT | 1 | Unit tests |
```

Comment:
do we want to be more granular than this? we'll probably support multiple types of "unit" tests in the future. to me it seems like we'd want to put each test implementation here, e.g. `structure` or `container-structure-test` (seeing as that's the only implementation we have atm)

Reply:
I added unit tests for now. I can add other types of tests. We can also revisit this once we finalize all the test types.

Reply:
BTW, the new flow is tested by the existing tests. Added more details in the description.